tools/mo/openvino/tools/mo/utils/telemetry_utils.py

```diff
@@ -103,3 +103,11 @@ def get_tid():
     This function returns the ID of the database to send telemetry.
     """
     return telemetry_params['TID']
+
+
```

Comment:
It is better to simulate a crash of TF FE by registering failing translator for some simple operation.

Reply:
This is up to you.